src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelTrace.General.cs

```diff
@@ -99,6 +104,9 @@ private static partial class GeneralLog
         [LoggerMessage(65, LogLevel.Warning, "HTTP/3 is not enabled for {Endpoint}. HTTP/3 requires TLS. Connections to this endpoint will use HTTP/1.1.", EventName = "Http3DisabledWithHttp1AndNoTls")]
         public static partial void Http3DisabledWithHttp1AndNoTls(ILogger logger, EndPoint endPoint);
 
+        [LoggerMessage(66, LogLevel.Debug, "The request has aborted.", EventName = "RequestAborted")]
```

Comment:
I also want the `@"Connection id ""{ConnectionId}"", Request id ""{TraceIdentifier}"":` here.